src/zope/configuration/xmlconfig.py

```diff
@@ -373,6 +373,23 @@ def evaluateCondition(self, expression):
                 return installed
             elif verb == 'not-installed':
                 return not installed
+
+        elif verb in ('envvar', 'not-envvar'):
+            if not arguments:
+                raise ValueError(
+                    "Environment variable name missing: %r" % expression
+                )
+            if len(arguments) > 1:
+                raise ValueError(
+                    "Only one environment variable name allowed: %r"
+                    % expression
+                )
```

Comment:
Do you see a chance to add coverage for these lines of code?
Currently the `coverage` test job breaks because of dropped coverage.

Reply:
Done.